ocp_install_env.sh

```diff
@@ -117,6 +125,7 @@ cat <<EOF
     provisioningBridge: ${PROVISIONING_NETWORK_NAME}
     provisioningNetworkCIDR: $PROVISIONING_NETWORK
     provisioningNetworkInterface: $CLUSTER_PRO_IF
+    bootstrapProvisioningInterfaceOverride: "${BOOTSTRAP_PROVISIONING_INTERFACE_OVERRIDE}"
```

Comment:
Looks like this depends on https://github.com/openshift/installer/pull/5554 - I'm wondering if there's a way to detect the required nic in the installer, vs expecting the user to know what the nic inside the (installer generated) bootstrap VM is?

Reply:
@jeffdyoung is working on the installer to make the detection of the interface automatic. I've removed this option for now from this PR.